src/physics.F90

```diff
@@ -53,7 +53,7 @@ subroutine collision(p)
     end if
 
     ! check for very low energy
-    if (p % E < 1.0e-100_8) then
+    if (p % E < MIN_ENERGY) then
```

Comment:
It is likely only used at that location, but it should be defined in
constants.F90, unless an errant merge removed it (the code has compiled
successfully for me many times).  This is not necessary for any
NDPP-specific functions, but a small clarification I ended up putting in
when I was doing some testing.

I was actually planning on squashing the commits to something more
manageable this weekend if you'd like to wait to have a clearer history to
examine.

Thanks!
Adam

On Fri, Feb 13, 2015 at 9:55 AM, Will Boyd <notifications@github.com> wrote:

> In src/physics.F90
> <https://github.com/mit-crpg/openmc/pull/309#discussion_r24668625>:
>
> > @@ -53,7 +53,7 @@ subroutine collision(p)
> >      end if
> >
> >      ! check for very low energy
> > -    if (p % E < 1.0e-100_8) then
> > +    if (p % E < MIN_ENERGY) then
>
> It appears that MIN_ENERGY isn't defined anywhere else in the code?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/309/files#r24668625>.
>